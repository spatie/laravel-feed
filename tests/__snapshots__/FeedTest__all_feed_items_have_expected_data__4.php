<?php return '<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0">
    <channel>
        <title><![CDATA[Feed 1 RSS]]></title>
        <link><![CDATA[http://localhost/feedBaseUrl/feed1.rss]]></link>
        <description><![CDATA[This is feed 1 as RSS from the unit tests]]></description>
        <language>en-US</language>
        <pubDate>Fri, 01 Jan 2016 00:00:00 +0100</pubDate>

                    <item>
                <title><![CDATA[feedItemTitle]]></title>
                <link>https://localhost/news/testItem1</link>
                <description><![CDATA[feedItemSummary]]></description>
                <author><![CDATA[feedItemAuthor]]></author>
                <guid>http://localhost/1</guid>
                <pubDate>Fri, 01 Jan 2016 00:00:00 +0100</pubDate>
            </item>
                    <item>
                <title><![CDATA[feedItemTitle]]></title>
                <link>https://localhost/news/testItem1</link>
                <description><![CDATA[feedItemSummary]]></description>
                <author><![CDATA[feedItemAuthor]]></author>
                <guid>http://localhost/1</guid>
                <pubDate>Fri, 01 Jan 2016 00:00:00 +0100</pubDate>
            </item>
                    <item>
                <title><![CDATA[feedItemTitle]]></title>
                <link>https://localhost/news/testItem1</link>
                <description><![CDATA[feedItemSummary]]></description>
                <author><![CDATA[feedItemAuthor]]></author>
                <guid>http://localhost/1</guid>
                <pubDate>Fri, 01 Jan 2016 00:00:00 +0100</pubDate>
            </item>
                    <item>
                <title><![CDATA[feedItemTitle]]></title>
                <link>https://localhost/news/testItem1</link>
                <description><![CDATA[feedItemSummary]]></description>
                <author><![CDATA[feedItemAuthor]]></author>
                <guid>http://localhost/1</guid>
                <pubDate>Fri, 01 Jan 2016 00:00:00 +0100</pubDate>
            </item>
                    <item>
                <title><![CDATA[feedItemTitle]]></title>
                <link>https://localhost/news/testItem1</link>
                <description><![CDATA[feedItemSummary]]></description>
                <author><![CDATA[feedItemAuthor]]></author>
                <guid>http://localhost/1</guid>
                <pubDate>Fri, 01 Jan 2016 00:00:00 +0100</pubDate>
            </item>
            </channel>
</rss>
';
