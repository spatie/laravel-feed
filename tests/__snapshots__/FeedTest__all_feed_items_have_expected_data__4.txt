<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0">
    <channel>
        <title><![CDATA[Feed 1 RSS]]></title>
        <link><![CDATA[http://localhost/feedBaseUrl/feed1.rss]]></link>
        <description><![CDATA[This is feed 1 as RSS from the unit tests]]></description>
        <language>en-US</language>
        <pubDate>2016-01-01T00:00:00+01:00</pubDate>

                    <item>
                <title><![CDATA[feedItemTitle]]></title>
                <link>https://localhost/news/testItem1</link>
                <description><![CDATA[feedItemSummary]]></description>
                <author><![CDATA[feedItemAuthor]]></author>
                <guid>http://localhost/1</guid>
                <pubDate>2016-01-01T00:00:00+01:00</pubDate>
                                    <category>feedItemCategory</category>
                            </item>
                    <item>
                <title><![CDATA[feedItemTitle]]></title>
                <link>https://localhost/news/testItem1</link>
                <description><![CDATA[feedItemSummary]]></description>
                <author><![CDATA[feedItemAuthor]]></author>
                <guid>http://localhost/1</guid>
                <pubDate>2016-01-01T00:00:00+01:00</pubDate>
                                    <category>feedItemCategory</category>
                            </item>
                    <item>
                <title><![CDATA[feedItemTitle]]></title>
                <link>https://localhost/news/testItem1</link>
                <description><![CDATA[feedItemSummary]]></description>
                <author><![CDATA[feedItemAuthor]]></author>
                <guid>http://localhost/1</guid>
                <pubDate>2016-01-01T00:00:00+01:00</pubDate>
                                    <category>feedItemCategory</category>
                            </item>
                    <item>
                <title><![CDATA[feedItemTitle]]></title>
                <link>https://localhost/news/testItem1</link>
                <description><![CDATA[feedItemSummary]]></description>
                <author><![CDATA[feedItemAuthor]]></author>
                <guid>http://localhost/1</guid>
                <pubDate>2016-01-01T00:00:00+01:00</pubDate>
                                    <category>feedItemCategory</category>
                            </item>
                    <item>
                <title><![CDATA[feedItemTitle]]></title>
                <link>https://localhost/news/testItem1</link>
                <description><![CDATA[feedItemSummary]]></description>
                <author><![CDATA[feedItemAuthor]]></author>
                <guid>http://localhost/1</guid>
                <pubDate>2016-01-01T00:00:00+01:00</pubDate>
                                    <category>feedItemCategory</category>
                            </item>
            </channel>
</rss>
